contracts/Builder/BuilderInternal.sol

```diff
@@ -6,7 +6,7 @@ import "@poolzfinance/poolz-helper-v2/contracts/interfaces/ISimpleProvider.sol";
 
 contract BuilderInternal is BuilderModifiers {
     ///@dev if params is empty, then return [amount]
-    function _concatParams(uint amount, uint256[] calldata params) internal pure returns (uint256[] memory result) {
+    function _concatParams(uint amount, uint256[] memory params) internal pure returns (uint256[] memory result) {
```

Comment:
I think better to have 2 functions, one with `calldata` and one with `memory`

Reply:
This will result in duplicate logic, but no problem 🤔